.github/workflows/cli.yml

```diff
@@ -5,10 +5,12 @@ on:
     branches:
       - master
       - v1
+      - v3-development-master
   pull_request:
     branches:
       - master
       - v1
+      - v3-development-master
```

Comment:
⭐ _(nice to have)_ can we name this just `v3`? For the sake of consistency 🙏

Reply:
Not a huge deal either way, but I actually like having a name like `v3-development-master` that indicates a lack of stability compared to `v3`.

Reply:
I agree with @rliebz it's just temporary. I will rename it once it's stable :)